supergloo/docs/mesh/tutorials-4-security.md

```diff
@@ -0,0 +1,287 @@
+? enter a comma-separated list of HTTP methods to allow for this rule, e.g.: GET,POST,PATCH (leave empty to allow all):
+? enter a comma-separated list of HTTP paths to allow for this rule, e.g.: /api,/admin,/auth (leave empty to allow all):
+```
+
+* The equivalent noninteractive version of this command would be
+
+```bash
+supergloo apply securityrule \
+    --name enable-security \
+    --namespace default \
+    --source-upstreams supergloo-system.default-productpage-9080 \
+    --dest-upstreams supergloo-system.default-productpage-9080 \
+    --target-mesh supergloo-system.istio
+```
+
+* Or, using `kubectl`:
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: supergloo.solo.io/v1
+kind: SecurityRule
+metadata:
+  name: enable-security
+  namespace: default
+spec:
+  destinationSelector:
+    upstreamSelector:
+      upstreams:
+      - name: default-productpage-9080
+        namespace: supergloo-system
+  sourceSelector:
+    upstreamSelector:
+      upstreams:
+      - name: default-productpage-9080
+        namespace: supergloo-system
+  targetMesh:
+    name: istio
+    namespace: supergloo-system
+EOF
+``` 
+
+Now that we've created our first SecurityRule, Access-Control should be enforced in our cluster.
+Now try refreshing the Product Page. 
+
+> Note: it may take up to a minute for configuration to propagate to all the Istio sidecars.
+
+We should see that the details and reviews sections of the UI have turned into error messages:
+
+![Bookinfo Product Page](../../img/bookinfo-all-error.png "Bookinfo Product Page")
+
+This confirms that the `productpage` service can no longer send requests to `details` or `reviews`.
+
+Let's add a rule to permit the `productpage` to communicate with `details`.
+
+* Using `supergloo` CLI:
+
+```bash
+supergloo apply securityrule \
+    --name productpage-to-details \
+    --namespace default \
+    --source-upstreams supergloo-system.default-productpage-9080 \
+    --dest-upstreams supergloo-system.default-details-9080 \
+    --target-mesh supergloo-system.istio
+```
+
+* Or with `kubectl`:
+
+```yaml
+cat  <<EOF | kubectl apply -f -
+apiVersion: supergloo.solo.io/v1
+kind: SecurityRule
+metadata:
+  name: productpage-to-details
+  namespace: default
+spec:
+  destinationSelector:
+    upstreamSelector:
+      upstreams:
+      - name: default-details-9080
+        namespace: supergloo-system
+  sourceSelector:
+    upstreamSelector:
+      upstreams:
+      - name: default-productpage-9080
+        namespace: supergloo-system
+  targetMesh:
+    name: istio
+    namespace: supergloo-system
+EOF
+```
+
+Refresh the Product Page again. We should see a page like this:
+
+![Bookinfo Product Page](../../img/bookinfo-details-enabled.png "Bookinfo Product Page")
+
+Communication has been enabled between `productpage` and `details`. Let's amend our rules to permit 
+`productpage` to reach any service in the `default` namespace:
+
+
+* Using `supergloo` CLI:
+
+```bash
+supergloo apply securityrule \
```

Comment:
This one didn't seem to have any effect either.

Reply:
did you try waiting several minutes?

Reply:
Yah. I can try again once quay comes back.